install/operator/.lib.sh

```diff
@@ -116,13 +116,22 @@ build_operator()
         echo building executable
         go test -test.short -mod=vendor ./cmd/... ./pkg/...
 
+        if [ -z "${GOOSLIST}" ]; then
+            GOOSLIST="linux darwin windows"
+        fi
+
         for GOARCH in amd64 ; do
-          for GOOS in linux darwin windows ; do
+          for GOOS in ${GOOSLIST} ; do
             export GOARCH GOOS
             echo building ./dist/${GOOS}-${GOARCH}/syndesis-operator executable
             go build  "$@" -o ./dist/${GOOS}-${GOARCH}/syndesis-operator \
                 -gcflags all=-trimpath=${GOPATH} -asmflags all=-trimpath=${GOPATH} -mod=vendor \
                 ./cmd/manager
+
+            echo building ./dist/${GOOS}-${GOARCH}/platform-detect executable
+            go build -o ./dist/${GOOS}-${GOARCH}/platform-detect \
```

Comment:
What is the use of this binary?

Reply:
platform-detect externalises the ApiCapabilities function for use in the syndesis cli scripts. You haven't seen those changes yet (haven't finalised them).